client/lib/utils.js

```diff
@@ -20,13 +20,14 @@ var utils = (function() {
         },
 
         /// Function: storeLocation()
-        /// Description: Stores longitude and latitude to the window. takes callback
+        /// Description: Stores longitude, latitude, and google maps link to the window. takes callback
         /// returns: Nothing
         storeLocation: function(cb, position) {
           this.latitude = position.coords.latitude;
           this.longitude = position.coords.longitude;
+          this.mapLink = 'http://maps.google.com/?q=' + this.latitude + ',' + this.longitude;
```

Comment:
Can you add a mapLink property to this object? Although this will populate it, it will prevent any issues arising from it not being defined.